update Activiti/activiti-cloud-build to develop

jx-activiti-cloud opened the pull request:
[UpdateBot](https://github.com/jenkins-x/updatebot) pushed version changes from the source code in repository: `Activiti/activiti-cloud-build` ref: `master`

ghost commented:
[UpdateBot](https://github.com/jenkins-x/updatebot) commands:

    updatebot push --ref master https://github.com/Activiti/activiti-cloud-build.git

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/Activiti/activiti-cloud-connectors?pullRequest=24) <br/>All committers have signed the CLA.

ghost commented:
[UpdateBot](https://github.com/jenkins-x/updatebot) commands:

    updatebot push --ref develop https://github.com/Activiti/activiti-cloud-build.git

Commit:
fix(versions): push Activiti/activiti-cloud-build ref: develop

Push version changes from the source code in repository: Activiti/activiti-cloud-build ref: develop
